### ISSUE TITLE
WCAG用語: 慣用句 (idiom) の見直し

### ISSUE THREAD

**Issue by jidaikobo-shibata** [waic/wcag22#1459]
ID: `dfn-idioms`
  この用語について検討します。
  [慣用句 (idiom)](https://waic.jp/translations/WCAG22/#dfn-idioms)

**Comment by momdo** [comment]
> 個々の単語の意味からはその意味を推測できず、特定の単語を変えると意味が通じなくなる言い回し。

> phrase whose meaning cannot be deduced from the meaning of the individual words and the specific words cannot be changed without losing the meaning

直訳すると「意味を失うことなく特定の意味を変えることができない」ですが、まあ意訳で大丈夫でしょう
